ipn/ipnlocal/local_test.go

```diff
@@ -1593,6 +1615,8 @@ func dnsResponse(domain, address string) []byte {
 }
 
 func TestSetExitNodeIDPolicy(t *testing.T) {
+	e := &tailcfg.Hostinfo{}
+	zeroValHostinfoView := e.View()
```

Comment:
Fine as-written, but could also be
```suggestion
	zeroValHostinfoView := new(tailcfg.Hostinfo).View()
```
if you don't need the intermediate for anything else. Up to you, though!